packages/components/forms/src/textarea/Textarea.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { cx } from 'emotion';
+
+import { BaseInput } from '../base-input';
+import type { BaseInputInternalProps } from '../base-input/types';
+import { getStyles } from './Textarea.styles';
+
+export type TextareaProps = Omit<BaseInputInternalProps, 'as'>;
```

Comment:
Can't we import the `BaseInputProps` directly? So it also has the types from the 'input' html tag?
Also maybe Omit the `type` property as it has no effect on the textarea component

Reply:
Yes we can

Reply:
Maybe we can't?
![Screenshot 2021-09-01 at 15 51 06](https://user-images.githubusercontent.com/6597467/131683516-aa13924b-8f88-4be5-bed8-595618197b55.png)

Reply:
But shouldn't the label be required anyway? Or `ariaLabel` as it's going to be renamed to: https://github.com/contentful/forma-36/pull/1266

Reply:
I'm not doing anything in Textarea
The component is using the BaseInputProps directly (well... omitting two props but still)

Isn't the error here saying that to `BaseInput` `label` is required but to `Textarea` (and therefore `BaseInputProps`) `label` is optional???

which means, `BaseInputProps` and `BaseInputInternalProps` are treating label differently and `BaseInputProps` does not reflect the real API of `BaseInput`

Reply:
Ohh got it, the `BaseInputProps` isn't exported from `types.ts`
either:
```js
import type { BaseInputProps } from '../base-input';
```
or
```js
import { BaseInput, BaseInputProps } from '../base-input';
 ```

Reply:
![Screenshot 2021-09-01 at 17 08 33](https://user-images.githubusercontent.com/6597467/131696446-a16c786e-5d61-4034-a7f5-8a9c9def528d.png)

There is still something to be done here

Reply:
How are you using the type?? it should be `Omit<BaseInputProps<'textarea'>, 'as' | 'type'>`

Reply:
voilá! I was missing the `<'textarea'>` in BaseInputProps